src/torcontrol.cpp

```diff
@@ -251,7 +251,7 @@ std::map<std::string,std::string> ParseTorReplyMapping(const std::string &s)
             /**
              * Unescape value. Per https://spec.torproject.org/control-spec section 2.1.1:
              *
-             *   For future-proofing, controller implementors MAY use the following
+             *   For future-proofing, controller implementers MAY use the following
```

Comment:
Actually, both `implementors` & `implementers` are valid words, but it seems they have different meaning (Implementer: One who has implemented, Implementor: One who provides implementation).